components/CoinGraphChart/index.tsx

```diff
@@ -2,7 +2,14 @@
 
 import { useAppSelector } from "@/redux/store";
 import { SelectedCoin } from "@/interfaces/selectedcoin.interface";
+import { getSelectedCoinData } from "@/redux/features/selectedCoins";
+import { AppDispatch } from "@/redux/store";
+import { useDispatch } from "react-redux";
+import { useEffect } from "react";
```

Comment:
Import files as follow. Absolute imports, components, utils/functions and finally styles.